Source/Engine/Session.cs

```diff
@@ -1,16 +1,87 @@
-﻿using System;
+﻿using Microsoft.EntityFrameworkCore;
+using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
 using System.Text;
 
 namespace EngineClasses
 {
     public class Session
     {
         [Key]
-        public int SessionId { get; set; }
-        public ICollection<Player> Player { get; set; }
+        public int SessionId { get; private set; }        
         public int Turns { get; set; }
+
+        //Relationships
+        public ICollection<Player> Player { get; private set; }
+
+        public Session()
+        {
+            Player = new List<Player>();
+        }
+
+        public void CreatePlayer(string userName, string color)
+        {
+            if (Player.Count < 4)
+            {
```

Comment:
Shouldn't it be <= 4 ?
What if it's more than should happen if there is more than 4? Should an exception be thrown?

Reply:
Slarvfel ser det ut som! Tack för kommentaren! Hade kunnat bli körigt att hitta.